src/MusicalScore/Graphical/MusicSheetCalculator.ts

```diff
@@ -166,7 +166,7 @@ export abstract class MusicSheetCalculator {
             );
             measureList.push(graphicalMeasures);
         }
-        this.handleStaffEntries();
+        this.handleStaffEntries(activeClefs);
```

Comment:
This works typically, but is logically not so perfect, as the active clefs will here be actually the final clefs after iterating through all measures before. Maybe it is better to convert these activeClefs (which are now final clefs) to a bool array:
staffIsPercussionArray. 
This would make it more clear.